add-ons/fabric8/README.md

```diff
@@ -0,0 +1,50 @@
+# Fabric8 Addon
+
+This addon provides an easy way to install fabric8 on MiniShift. 
+
+Fabric8 provides a complete developer platform for creating projects, editing them and performing full CI / CD for the entire developer lifecycle.
+
+## Setup GitHub OAuth client ID and secret
+
+Fabric8 has GitHub integration letting you browse repositories, create new repositories, edit projects and setup automated CI / CD jobs with webhooks on github. 
+
+This requires an [OAuth application to be setup on your github account](https://developer.github.com/apps/building-integrations/setting-up-and-registering-oauth-apps/registering-oauth-apps/) for fabric8 and you need to obtain the client ID and secret for the OAuth application.
```

Comment:
I'm hoping we can make the UI self describable if its confusing ;). Right now when you click the Login button you are presented with a user/pwd for KeyCloak if you have an account or a button to login to OpenShift (other identity providers can be configured).

Here's the issue https://github.com/fabric8io/fabric8-platform/issues/683 - am hoping we can polish that page to make it more self explanatory